### PR TITLE
Export with log-softmax and softmax of logits

### PR DESCRIPTION
**Pull request by rsuderman**
For some searches (sampling, beamsearch) it is beneficial to return either the softmax or log softmax of the values.

### CODE REVIEW

File: sharktank/sharktank/examples/export_paged_llm_v1.py [hunk 1]
```diff
@@ -132,6 +140,7 @@ def generate_params_json(
             "prefill_batch_sizes": prefill_bs,
             "decode_batch_sizes": decode_bs,
             "transformer_block_count": hp.block_count,
+            "logits_normalization": logits_normalization,
```
Comment (stbaione) [inline sharktank/sharktank/examples/export_paged_llm_v1.py:143]: Need to add a field to [ModelParams](https://github.com/nod-ai/shark-ai/blob/main/shortfin/python/shortfin_apps/llm/components/config_struct.py#L106)

[Failing Test](https://github.com/nod-ai/shark-ai/actions/runs/14202182333/job/39791620157?pr=1196#step:5:96536)

Comment (rsuderman) [reply sharktank/sharktank/examples/export_paged_llm_v1.py:143]: Updated.

File: sharktank/sharktank/examples/export_paged_llm_v1.py [hunk 2]
```diff
@@ -264,6 +273,12 @@ def _(model, tokens, seq_lens, seq_block_ids, cs):
             if llama_config.tensor_parallelism_size != 1:
                 logits = ops.unshard(logits)
 
+            if args.logits_normalization == "softmax":
```
[the remark this answers is not in the document]
Comment (rsuderman) [reply sharktank/sharktank/examples/export_paged_llm_v1.py:276]: It is possible (though not possible to be optional). I want to avoid making any changes on the API as it will just cause a host of headaches for this week.

[the remark this answers is not in the document]
Comment (stbaione) [reply sharktank/sharktank/examples/export_paged_llm_v1.py:276]: Sounds good

**Review by stbaione — APPROVED** [review]
LGTM, should make an issue to get things like perplexity tests setup for the `softmax` and `log_softmax` options
